packages/svelte-ux/src/lib/utils/dateRange.ts

```diff
@@ -1,35 +1,47 @@
 import { startOfDay, isLeapYear, isAfter, isBefore, subYears } from 'date-fns';
 
-import { getDateFuncsByPeriodType } from './date';
+import { getDateFuncsByPeriodType, updatePeriodeTypeWithWeekStartsOn } from './date';
 import { PeriodType } from './date_types';
+import type { LocaleSettings } from '.';
 
 export type DateRange = {
   from: Date | null;
   to: Date | null;
   periodType?: PeriodType | null;
 };
 
-export const dayPresets = getDateRangePresets(PeriodType.Day);
-export const biWeekSun1Presets = getDateRangePresets(PeriodType.BiWeek1Sun);
-export const biWeekMon1Presets = getDateRangePresets(PeriodType.BiWeek1Mon);
-export const monthPresets = getDateRangePresets(PeriodType.Month);
-export const quarterPresets = getDateRangePresets(PeriodType.Quarter);
-export const fiscalYearPresets = getDateRangePresets(PeriodType.FiscalYearOctober);
+// All this is exported, but never used. Is it usefull?
```

Comment:
I thought I used these for something, but I checked a few of my other projects and didn't find any.  We could probably remove the `export`